lib/s.ml

```diff
@@ -53,8 +53,8 @@ module type SERVER = sig
   (** The name of an export. In the 'new style' protocol as used in nbd >= 2.9.17
       the client must select an export by name. *)
 
-  val connect : channel -> ?offer:name list -> unit -> (name * t) Lwt.t
-  (** [connect channel ?offer ()] performs the 'new style' initial handshake
+  val connect : cleartext_channel -> ?offer:name list -> ?no_tls:bool -> unit -> (name * t) Lwt.t
```

Comment:
The `?no_tls` parameter is optional to preserve backward compatibility with older clients.
Do we need backward compatibility with older clients? If not, then we can make this parameter mandatory. Ping @djs55